helm-limits/requests-and-limits.yaml

```diff
@@ -0,0 +1,136 @@
+# values for limits can vary greatly depending on configuration and number of containers monitored.
+# We suggesting letting Kubecost run for a few days at full load and then use recommendations from the "Right-size your container requests" report in "Savings"
+
+kubecostFrontend:
+  resources:
+    requests:
+      cpu: 10m
+      memory: 55Mi
+  # limits:
+  #   cpu: 100m
+  #   memory: 256Mi
+
+kubecostMetrics:
+  exporter:
+    resources:
+       requests:
+        cpu: 200m
+        memory: 55Mi
+     # limits:
+     #  cpu: 200m
+     #  memory: 55Mi
+
+kubecostModel:
+  resources:
+    requests:
+      cpu: 200m
+      memory: 55Mi
+  # limits:
```

Comment:
Should we add a higher example limit here and comment on what typical limit for big cluster should look like? e.g: Clusters with x pods would need yy CPU and zz Memory.

Reply:
yes! 

I'll merge, feel free to make it better